README.md

```diff
@@ -35,7 +35,7 @@ The `*.min.*` scripts are minified.
         new MediumEditor(document.querySelector(".editor"), {
             extensions: {
                 markdown: new MeMarkdown(function (md) {
-                    markDownEl.innerText = md;
+                    markDownEl.innerHTML = md;
```

Comment:
Does this work smoothly if you write `<script>alert()</script>` in the editor? If so, that's awesome, otherwise maybe we should use `innerContent` (or something like that).